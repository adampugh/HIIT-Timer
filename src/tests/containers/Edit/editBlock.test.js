import React from "react";
import { shallow } from "enzyme"; 

import EditBlock from "../../../containers/Edit/editBlock";
import workoutState from "../../fixtures/workouts";

test("should render editBlock correctly", () => {
    const exercise = workoutState.workouts[0].exercises[0];
    const wrapper = shallow(<EditBlock 
        name={exercise.name} 
        time={exercise.time} />);
    expect(wrapper).toMatchSnapshot();
});