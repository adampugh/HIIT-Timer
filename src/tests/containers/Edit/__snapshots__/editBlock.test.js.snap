// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render editBlock correctly 1`] = `
<div
  className="editBlock"
  style={
    Object {
      "background": undefined,
    }
  }
>
  <h1>
    squats
  </h1>
  <h1>
    01:00
  </h1>
</div>
`;
